src/player.js

```diff
@@ -2287,7 +2287,18 @@ export default class Player extends FakeEventTarget {
    */
   _onCueChange(event: FakeEvent): void {
     Player._logger.debug('Text cue changed', event.payload.cues);
-    this._activeTextCues = event.payload.cues;
+    //TODO: remove filter once FEC-11048 fix is done
+    try {
+      this._activeTextCues = event.payload.cues.filter((cue, index, cues) => {
+        const prevCue = cues[index - 1];
+        if (!prevCue) {
+          return true;
+        }
+        return !(cue.startTime === prevCue.startTime && cue.endTime === prevCue.endTime && cue.text.trim() === prevCue.text.trim());
```

Comment:
`cue.startTime === prevCue.startTime && cue.endTime === prevCue.endTime` must be true if you get list of cues right?

Reply:
yes, the times must be the same, only diff is text which is not trimmed between cues.